src/ui/components/SecondaryToolbox/SecondaryToolbox.css

```diff
@@ -112,5 +112,5 @@ overflows its container (https://drafts.csswg.org/css-flexbox-1/#min-size-auto)
   left: -2rem;
   width: 2rem;
   height: 100%;
-  background-image: linear-gradient(to left, rgba(0, 0, 0, 1), rgba(0, 0, 0, 0));
+  background-image: linear-gradient(to left, var(--tab-bgcolor), rgba(0, 0, 0, 0));
```

Comment:
I think fading from white to `rgba(0, 0, 0, 0)` will cause not just the opacity but the color to fade.

Reply:
Hm -- it feels pretty good to me, am I missing something?

Reply:
Interesting! I wonder if different browsers treat this differently. Or maybe I'm mistaken on how the fade works when there is a color _and_ opacity shift! My apologies.